### DISCUSSION TITLE
Raffle Drawing or General Drawing Only

### DISCUSSION THREAD

**Discussion by jacywest** [MCLD/greatreadingadventure#997] [Help!]
We're setting up the GRA for my library (THANK YOU for this option), and I'm getting to know the different features. For drawings, is it only an unspecified, anyone who has x points at x location can win a drawing, or is there a way to have a raffle-like set up where patrons put their name in a specific drawing for a prize they specifically want?

**Comment by k7hpn** [comment]
Hi @jacywest! We don't currently have a 'raffle'-type feature as you describe, though it sounds like a good idea. I know we have had the situation in our program where staff have called a winner and allowed the winner to select one of a few prizes - so at least they get some say in what they've won. I've entered #1001 as a feature request for this though I don't know when we will get to it. Thanks for asking and sharing the idea! Let me know if you have any other comments/questions/concerns!!

**Reply by jacywest** [reply]
Thank you for looking into it as a feature! For summer reading patrons can pick digital logging or physical, so I'll make everyone use physical tickets for now (show a badge, get a ticket), but for winter its all digital so I appreciate the information on how others handle multi-prize drawings.
